src/config/icons.ts

```diff
@@ -0,0 +1,47 @@
+import { ARBITRUM, AVALANCHE } from "config/chains";
+import arbitrum from "img/ic_arbitrum_24.svg";
+import avalanche from "img/ic_avalanche_24.svg";
+import gmxIcon from "img/ic_gmx_40.svg";
+import glpIcon from "img/ic_glp_40.svg";
+import gmxArbitrum from "img/ic_gmx_arbitrum.svg";
+import gmxAvax from "img/ic_gmx_avax.svg";
+import glpArbitrum from "img/ic_glp_arbitrum.svg";
+import glpAvax from "img/ic_glp_avax.svg";
+
+type Icons = {
+  [ARBITRUM]: {
+    network: string;
+    gmx: string;
+    glp: string;
+  };
+  [AVALANCHE]: {
+    network: string;
+    gmx: string;
+    glp: string;
+  };
+  gmx: string;
+  glp: string;
+};
+
+const ICONS: Icons = {
+  [ARBITRUM]: {
+    network: arbitrum,
+    gmx: gmxArbitrum,
+    glp: glpArbitrum,
+  },
+  [AVALANCHE]: {
+    network: avalanche,
+    gmx: gmxAvax,
+    glp: glpAvax,
+  },
+  gmx: gmxIcon,
+  glp: glpIcon,
+};
+
+export function getIcons(network, label) {
```

Comment:
1. i think `getIcon` not `getIcons` ? Because this function returns only one icon?

2. the config structure looks inconsistent because of common glp and gmx icon
3. in some cases we need to specify the size
4. There is CHAIN_ICONS in config/chains i think wee need to store it in one place
5. we don't need to define the type here, because  typescript should infer this automatically

about 2,3,4, 5 as a possible solution with type checking
```
const ICONS = {
  [ARBITRUM]: {
    network: {
      30: "ic_arbitrum_30.svg",
      16: "ic_arbitrum_16.svg",
    },
    glp: {
      30: "ic_arbitrum_glp_30.svg",
    }
  },
  [AVALANCHE]: {
    network: {
      30: "ic_avalanche_30.svg",
      16: "ic_avalanche_16.svg",
    },
    glp: {
      30: "ic_avalanche_glp_30.svg",
    }
  },
  common: {
    glp: {
      30: "ic_glp_30.svg",
    }
  },
};

type Icons = typeof ICONS;

function getIcon<
  TChain extends keyof Icons,
  TName extends keyof Icons[TChain],
  TSize extends keyof Icons[TChain][TName]
>(chainId: TChain, name: TName, size: TSize) {}

const arb = getIcon(ARBITRUM, "network", 30);
const glp = getIcon('common', "glp", 30);
```

Reply:
Would wait for comments from @gdev8317

Reply:
I feel we don't need to have multiple sizes of the icons for `svg` files. Scalability is a huge advantage of using svg. We can have one icon and add the width attributes to the `img` element.

The issue with the sizes is it can't be standard for each icons and situations. Please let me know your thoughts on this.

Reply:
Ah, yes, you're right about the SVG, the sizes aren't needed here then

Reply:
Just now in the project for some reason we keep several sizes for svg icons, maybe we should make them all the same default size?